File: model/ConfigHelper.vb
Imports System.Configuration

Public Class ConfigHelper
    Public Shared Function GetConnectionString(ByVal name As String) As String
        Return ConfigurationManager.ConnectionStrings(name).ConnectionString
    End Function

    Public Shared Function GetAppSetting(ByVal key As String) As String
        Return ConfigurationManager.AppSettings(key)
    End Function

End Class

File: model/ConfigHelper.cs
using System.Configuration;

namespace SchemaZen.Library.Models {
	public class ConfigHelper {
		public static string GetConnectionString(string name) {
			return ConfigurationManager.ConnectionStrings[name].ConnectionString;
		}

		public static string GetAppSetting(string key) {
			return ConfigurationManager.AppSettings[key];
		}
	}
}
